tsconfig.json

```diff
@@ -20,7 +20,10 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "paths": {
+      "@uikit/*": ["./packages/uikit/src/*"],
+    }
```

Comment:
what is path for?

Reply:
Related to the thing I mentioned in the key change. 

>what is path for?
 
To enable this `@uikit/***`.

Reply:
so this is necessary when using submodule?